apis/core/v1beta1/featureflaginprocessconfiguration_types.go

```diff
@@ -0,0 +1,195 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"fmt"
+
+	"github.com/open-feature/open-feature-operator/apis/core/v1beta1/common"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// FeatureFlagInProcessConfigurationSpec defines the desired state of FeatureFlagInProcessConfiguration
+type FeatureFlagInProcessConfigurationSpec struct {
+	// Port defines the port to listen on, defaults to 8013
+	// +kubebuilder:default:=8013
```

Comment:
not related to this PR but should we move the default to 8080 on flagd since it's HTTP traffic? 🤔 @toddbaert @Kavindu-Dodan

Reply:
I might be confused but I still think this is gRPC traffic.

Reply:
I confused myself between ports 🤦 8015 is the one for OFREP and that's the one I was thinking to move to 8080

Reply:
Sorry for late review, OFREP runs on 8016 . And gRPC sync service runs on 8015 [1]  (8013 is for remote evaluations).

Having dedicated ports is confusing but also gives users ability to control deployment at network level.

Anyway this port refers to gRPC sync port, so the default should be 8015 :) 

[1] - https://github.com/open-feature/flagd/blob/main/flagd/cmd/start.go#L41-L45

Reply:
IMO we should make default port to `8015` so users can use flagd itself as a gRPC sync source with `Flagd` crd.